src/plone/testing/z2.py

```diff
@@ -543,6 +543,8 @@ def setUpDatabase(self):
         self['zodbDB'] = zodb.stackDemoStorage(
             self.get('zodbDB'),
             name='Startup')
+        import transaction
+        transaction.begin()
```

Comment:
This will abort the current transaction if there was already one. I am not sure if this was intended.